src/locales/en_US.js

```diff
@@ -26,6 +26,125 @@ const strings = {
   exchange_failed: 'Exchange Failed',
   exchange_succeeded: 'Exchange Succeeded',
   exchanges_may_take_minutes: 'Exchanges may take several minutes to process. Please check your destination wallet after a few minutes',
+  bitid_identity: 'BitID Identity',
```

Comment:
This looks like a merge issue. These strings shouldn't have all been added back. Most were moved to the en_unused.js file